src/main/java/io/vertx/core/http/HttpServerOptions.java

```diff
@@ -681,6 +721,96 @@ public HttpServerOptions setDecoderInitialBufferSize(int decoderInitialBufferSiz
     return this;
   }
   
+  /**
+   * Enable or disable support for WebSocket Defalte Frame compression
+   * @param deflateCompressionSupported
+   * @return a reference to this, so the API can be used fluently
+   */
+  public HttpServerOptions setIsEnabledWebsocketFrameDeflateCompression (boolean deflateCompressionSupported) {
```

Comment:
shouldn't it rather be `setWebsocketFrameDeflateCompressionSupported`, same for other getter/setter